ui/app/manager/src/pages/page-map.ts

```diff
@@ -341,33 +370,66 @@ export class PageMap extends Page<MapStateKeyed> {
 
     protected render() {
 
-        return html`
+        let currentAssetSelected: boolean = (this._currentAsset != undefined);
+        let isMouseAPointer = false;
+        if(this._map?._map?._mapGl != undefined){
+            isMouseAPointer = this._map._map._mapGl.getCanvas().style.cursor == '';
+        }
+        // currentAssetSelected? console.log("render") : console.log("asset unselected");
+            return html`
+<!--                
+                <or-map-location-history-markers .assetId="${this._currentAsset?.id}" .map="${this._map}"></or-map-location-history-markers>
+            -->
+                <or-map-location-history-overlay .assetId="${this._currentAsset?.id}" .map="${this._map}" ></or-map-location-history-overlay>
```

Comment:
If this location history is to be integrated into the generic manager app then it will need to be configurable